components/ai_chat/resources/page/components/conversations_list/index.tsx

```diff
@@ -166,36 +166,44 @@ export default function ConversationsList(props: ConversationsListProps) {
     <>
       <div className={styles.scroller}>
         <nav className={styles.nav}>
-          {!aiChatContext.isStoragePrefEnabled &&
-          <Alert type='notice'>
-            <Icon name='history' slot='icon' />
-            <div slot='title'>{getLocale('noticeConversationHistoryTitleDisabledPref')}</div>
-            {getLocale('noticeConversationHistoryDisabledPref')}
-            <div slot='actions'>
-              <Button kind='outline' onClick={aiChatContext.enableStoragePref}>
-                {getLocale('noticeConversationHistoryDisabledPrefButton')}
-              </Button>
-            </div>
-          </Alert>
-          }
-          {aiChatContext.isStoragePrefEnabled && aiChatContext.visibleConversations.length === 0 &&
-          <Alert type='notice'>
-            <Icon name='history' slot='icon' />
-            <div slot='title'>{getLocale('menuConversationHistory')}</div>
-            {getLocale('noticeConversationHistoryEmpty')}
-          </Alert>
-          }
-          {aiChatContext.visibleConversations.length > 0 &&
-          <ol>
-            {aiChatContext.visibleConversations.map(conversation =>
-              <ConversationItem
-                key={conversation.uuid}
-                {...props}
-                conversation={conversation}
-              />
-            )}
-          </ol>
-          }
+          {!aiChatContext.isStoragePrefEnabled && (
+            <Alert type='notice'>
+              <Icon name='history' slot='icon' />
+              <div slot='title'>
+                {getLocale('noticeConversationHistoryTitleDisabledPref')}
+              </div>
+              {getLocale('noticeConversationHistoryDisabledPref')}
+              <div slot='actions'>
+                <Button kind='outline'
+                        onClick={aiChatContext.enableStoragePref}>
+                  {getLocale('noticeConversationHistoryDisabledPrefButton')}
+                </Button>
+              </div>
+            </Alert>
+          )}
+          {aiChatContext.isStoragePrefEnabled &&
+            aiChatContext.visibleConversations.length === 0 && (
+            <Alert type='notice'>
+              <Icon name='history' slot='icon' />
+              <div slot='title'>
+                {getLocale('menuConversationHistory')}
+              </div>
+              {getLocale('noticeConversationHistoryEmpty')}
+            </Alert>
+          )}
+          {aiChatContext.visibleConversations.length > 0 && (
+            <ol>
+              {aiChatContext.visibleConversations
+                .filter(conversation => !conversation.temporary)
```

Comment:
This is the only change in this file, others in this file is just fixing presubmit warning.